recipes/numba_quaternion/meta.yaml

```diff
@@ -1,44 +0,0 @@
-{% set name = "numba_quaternion" %}
```

Comment:
Shouldn't be modifying files outside your recipe...

Reply:
Oh weird? Must be a botched merge conflict, I'll see if I can rebase that away

Reply:
@dopplershift how about now?

Reply:
👍

Reply:
Thanks for all your help @dopplershift, I wouldn't know what to do otherwise!